opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -425,6 +416,13 @@ export const addUser = async (context, user, newUser) => {
   if (existingUser) {
     throw FunctionalError('User already exists', { email: userEmail });
   }
+  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES)) {
+    // user is Organization Admin
+    const groupIds = R.uniq(user.administrated_organizations.map((orga) => orga.grantable_groups).flat());
+    if (!newUser.groups.every((group) => groupIds.includes(group))) {
+      throw ForbiddenAccess();
```

Comment:
don't we check that the `newUser` is also part of an administrated organization ? I see only a check on grantable groups, but I guess we could check first the organizations then test these organizations grantable groups

Reply:
also how does it behave if newUser.groups is empty ? can an admin orga create a new user without any group ? and any orga ? (via the API)